test/test_generate.jl

```diff
@@ -49,16 +49,30 @@ using Test
         @test !isempty(output)
     end
 
+    @testset "cutoff at max_steps" begin
+        output = generate(
+            transformer,
+            tokenizer,
+            sampler_1,
+            "",
+            verbose=false, 
+            display_output=false, 
+            display_prompt=false,
+            max_steps=10,
```

Comment:
Please stay consistent whether you use the optional comma after the last parameter or not.
Check every call of the generate function in the test file.